packages/core/demo/data/treemap.ts

```diff
@@ -0,0 +1,97 @@
+export const treemapData = [
+	{
+		name: "Oceania",
+		children: [
```

Comment:
Open question: are arbitrarily nested children allowed? I'm not super familiar with treemap variants, so I don't know if that would be desirable.

Reply:
you could provide those children, however we haven't designed for that yet